javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -163,6 +240,25 @@ class Throwable(s: String, private var e: Throwable)
     duplicates
   }
 
+  def setStackTrace(stackTrace: Array[StackTraceElement]): Unit = {
+
```

Comment:
The blank line here and at the end of the method are a tad too much, even for me who uses plenty of blank lines ;)

Reply:
I swept through both files and removed blank lines after open-curly and before
close-curly.  There was one place in the Scala.js original which had a blank
line after the open-curly, so I did not change that.

I will use this style in code I am writing or changing from now on, including
reviewed files.